### PR TITLE
Add "1m" timespan in Nextstrain profiles

### PR DESCRIPTION
**Pull request by victorlin**
## Description of proposed changes

_Motivated by [this Slack thread](https://bedfordlab.slack.com/archives/CSKMU6YUC/p1665755551606909)._

Everything is copied from entries defining the existing "2m" timespan. Comments and other timespan references updated accordingly.

## Preview staging builds

Note that these won't be ready until the trial runs have finished uploading.

- [gisaid/global/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/global/1m)
- [gisaid/africa/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/africa/1m)
- [gisaid/asia/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/asia/1m)
- [gisaid/europe/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/europe/1m)
- [gisaid/north-america/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/north-america/1m)
- [gisaid/oceania/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/oceania/1m)
- [gisaid/south-america/1m](https://nextstrain.org/staging/ncov/gisaid/trial/1m-timespan/south-america/1m)
- [open/global/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/global/1m)
- [open/africa/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/africa/1m)
- [open/asia/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/asia/1m)
- [open/europe/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/europe/1m)
- [open/north-america/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/north-america/1m)
- [open/oceania/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/oceania/1m)
- [open/south-america/1m](https://nextstrain.org/staging/ncov/open/trial/1m-timespan/south-america/1m)

## Related issue(s)

_N/A_

## Testing

- [x] Start trial runs from c220e1d7d228819a0b1f6c42ba5f27106003b9b6
    - [GISAID](https://github.com/nextstrain/ncov/actions/runs/3387921099)
    - [open](https://github.com/nextstrain/ncov/actions/runs/3387921099)

## Release checklist

~If this pull request introduces backward incompatible changes, complete the following steps for a new release of the workflow~ N/A

If this pull request introduces new features, complete the following steps:

 - [x] Update `docs/src/reference/change_log.md` in this pull request to document these changes by the date they were added.

### CODE REVIEW

**Comment by corneliusroemer** [comment]
Here are the job ids so one can check logs, open should be done by now but isn't...

```
nextstrain build --aws-batch --no-download --attach a3447a21-283d-4439-aef0-4f837dbcc89e .
nextstrain build --aws-batch --no-download --attach e64c991b-5d71-4eab-853a-9f1b757f1cf5 .
```

**Review by corneliusroemer — CHANGES_REQUESTED** [review]
Both trial runs unfortunately failed, see comment

**Comment by victorlin** [comment]
Thanks for digging into that @corneliusroemer. I'm about to sign out for the day, so feel free to make changes on the PR branch to your desire. Otherwise I can pick up on it tomorrow.

**Comment by victorlin** [comment]
As @trvrb [mentioned in Slack](https://bedfordlab.slack.com/archives/CSKMU6YUC/p1667434685478259?thread_ts=1665755551.606909&cid=CSKMU6YUC), logistic growth is [set to 6 weekly pivots](https://github.com/nextstrain/ncov/blob/c220e1d7d228819a0b1f6c42ba5f27106003b9b6/defaults/parameters.yaml#L161) which is too much when frequencies is limited by `--min-date 1M`. I've pushed c220e1d7d228819a0b1f6c42ba5f27106003b9b6 which should fix this. Will run new trial builds.

**Comment by victorlin** [comment]
Also @corneliusroemer it's the same error for both GISAID and open.

**Comment by corneliusroemer** [comment]
Excellent, now it works. I think we can make the bucket size bigger to get more recent sequences for the open build. We only have 1.3k sequences from last 1M and 2k sequences in total.

A build with 4k sequences is a good size, so we can add another 2k sequences from the most recent month to make it as useful as possible.

**Comment by victorlin** [comment]
@corneliusroemer I've just rebased onto latest master so it has your changes to upload builds as they are done. I think it'd be best if you take over this branch now and tweak to your desire. Would you be willing to do that?

File: docs/src/reference/change_log.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ We also use this change log to document new features that maintain backward comp
 
 ## New features since last version update
 
+- X November 2022: Add "2m" timespan in Nextstrain profile builds. [PR 1027](https://github.com/nextstrain/ncov/pull/1027)
```
Comment (corneliusroemer) [inline docs/src/reference/change_log.md:8]:
```suggestion
- 8 November 2022: Add "1m" timespan in Nextstrain profile builds. [PR 1027](https://github.com/nextstrain/ncov/pull/1027)

```

File: docs/src/reference/change_log.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ We also use this change log to document new features that maintain backward comp
 
 ## New features since last version update
 
+- X November 2022: Add "2m" timespan in Nextstrain profile builds. [PR 1027](https://github.com/nextstrain/ncov/pull/1027)
+
+- X November 2022: calculate_delta_frequency: Allow script to work with fewer pivots available than requested with `--delta-pivots`. [PR 1027](https://github.com/nextstrain/ncov/pull/1027)
```
Comment (corneliusroemer) [inline docs/src/reference/change_log.md:10]:
```suggestion
- 8 November 2022: calculate_delta_frequency: Allow script to work with fewer pivots available than requested with `--delta-pivots`. [PR 1027](https://github.com/nextstrain/ncov/pull/1027)

```

**Review by corneliusroemer — APPROVED** [review]

**Review by rneher — APPROVED** [review]

**Comment by victorlin** [comment]
@corneliusroemer FYI, I'm deleting your branch `very-recent-1m` at ca23d68ef2abf1b5b7eb5b2551b135840f39a627 (it looks like an old test branch related to this PR).

**Pull request merged**